LibreNMS/OS/Routeros.php

```diff
@@ -265,6 +266,18 @@ private function discoverSensor($type, $oid, $num_oid_base): array
                     'SSID: ' . $entry['mtxrWlApSsid'],
                     $entry[$oid]
                 );
+            } elseif (($entry['mtxrLTEModemInterfaceIndex'] !== null)) {
+                $intname = snmp_get($this->getDeviceArray(),
```

Comment:
Use `$this->getCacheTable()` instead of many snmp_get() to fetch mtxrInterfaceStatsName for a significant speedup.